usecases/data_model_usecase.go

```diff
@@ -114,34 +122,35 @@ func (usecase *DataModelUseCase) CreateDataModelTable(ctx context.Context, organ
 			if err := usecase.organizationSchemaRepository.CreateSchemaIfNotExists(ctx, orgTx); err != nil {
 				return err
 			}
-			return usecase.organizationSchemaRepository.CreateTable(ctx, orgTx, name)
+			if err := usecase.organizationSchemaRepository.CreateTable(ctx, orgTx, name); err != nil {
```

Comment:
It looks like in the whole data_model usecases file, we do not check for permission either, same question as above

Reply:
I'm not sure to see what you mean here, every usecase method from dataModelRepository has a permission check (usually towards the beginning).